### DISCUSSION TITLE
189天翼云(账号密码版) 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#46] [模板讨论]
189天翼云(账号密码版) 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by jerrell851** [comment]
<img width="882" alt="微信截图_20230626142301" src="https://github.com/wjf0214/qd-templates/assets/61078121/a2d586bc-36d1-40a9-ab58-b27bddd6214e">
自6月22日后就用不起了，一直提示这个错误。

**Reply by wjf0214** [reply]
这是你连不上做数据处理的那个网站

**Reply by wjf0214** [reply]
更改了使用内置API处理，可以更新下。
另外QD框架版本需大于20230626, 需要自行更新到dev版本。或者等待后续更新到稳定版。

**Reply by duby2010** [reply]
大佬，我用的[a76yyyy](https://github.com/a76yyyy)的签到，部署在群晖docker里面的，原来用得好好的，最近重装群晖后也有这个问题。请问这个环境下如何查看QD框架版本，需要怎么更新了？模板我是强制更新后才下载的。谢谢

**Reply by duby2010** [reply]
提示：
Failed at 6/20 request,
Fail assert: {"re": "\"状态\": \"OK\"", "from": "content"} from success_asserts,
Response Error : HTTP 500: The error occurred when rendering template data: r=json&p=&s={{unicode(b2a_hex(a2b_base64(userrsakey), sep=' ', bytes_per_sep=1))|urlencode}}&t=
TypeError("'str' object is not callable"),
Request URL: api://util/string/replace

**Reply by wjf0214** [reply]
@duby2010 框架首页顶部有显示。大于20230626就行。没有就更新docker项目。

**Comment by xc569643519** [comment]
![IMG_20230708_005521](https://github.com/wjf0214/qd-templates/assets/52149847/2debd289-5cec-446c-bef3-703c49bdfe2f)
我的就测试的时候成功了，之后就一直报错这个

**Comment by KoWming** [comment]
大佬a2b和b2a值怎么填？是账号和密码转换对应的编码吗？

**Reply by wjf0214** [reply]
就只有账号和密码。这2个值是要更新到最新版本的框架才能用。起码是20230626之后的版本。

**Reply by KoWming** [reply]
好的，感谢大佬！

**Comment by hecarli555** [comment]
今天突然提示这个了
<html>
<body>


Failed at 3/20 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 302: Moved Temporarily,Request URL: https://open.e.189.cn/api/logbox/oauth2/appConf.do
--



</body>
</html>

**Reply by wjf0214** [reply]
复现不出，可以再试试，看是不是网站调整导致的偶发事件。

**Reply by hecarli555** [reply]
原来是网站问题，后面自行恢复了，感谢大佬回复

**Comment by 8MiYile** [comment]
```
2023-11-11 10:47:31
日志：Failed at 5/20 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Failed to connect to localhost port 8923: Connection refused,
Request URL: api://util/rsa?f=encode&key=-----BEGIN%20PUBLIC%20KEY-----{{pubKey|urlencode}}-----END%20PUBLIC%20KEY-----&data={{username|urlencode}}
```

老哥这个报错是怎么回事啊

**Reply by wjf0214** [reply]
你是不是开了全局代理之类的，框架本身的内置接口访问失败了

**Reply by 8MiYile** [reply]
哦哦 这个我刚才解决了 我以为是你提供的程序有问题 原来是主程序的机制有问题
我代理在路由器上不在服务器

**Comment by ErDongChen1973** [comment]
现在用最新的签到程序也不能正常签到，请教一下是什么问题呢？
程序版本：
![image](https://github.com/wjf0214/qd-templates/assets/44294575/2fe1ffdc-7f48-4308-8c9c-70758df7f796)
错误信息：
![image](https://github.com/wjf0214/qd-templates/assets/44294575/83dfe5dc-c788-45ef-a890-cbbd7df2ff04)

**Reply by wjf0214** [reply]
有框架的运行日志吗？这个目前看不出来什么，看着是框架本身的内置接口出问题了。可能要借助框架的日志来排除下问题了。。或者再运行试试，是框架偶发性的bug问题还是怎么。。

**Reply by ErDongChen1973** [reply]
我更新版本后，首次运行这个，下面是日志：
![image](https://github.com/wjf0214/qd-templates/assets/44294575/1ae69783-3eef-4520-8b75-0424bd4da3a2)

**Reply by wjf0214** [reply]
目前还是不太好确定，获取不到相关具体信息。不过看着你是用的主库的模板，可以从主库的readme里看介绍，进QQ群问问框架作者(我已咨询，不过沟通上还不是很方便)，看看是框架问题还是什么，更方便沟通。
目前看就是未知原因导致文本转换的api请求失败了，可能要临时开debug模式看看更具体的日志给框架作者看下，我这目前是复现不出来。。

**Reply by ErDongChen1973** [reply]
> 目前还是不太好确定，获取不到相关具体信息。不过看着你是用的主库的模板，可以从主库的readme里看介绍，进QQ群问问框架作者(我已咨询，不过沟通上还不是很方便)，看看是框架问题还是什么，更方便沟通。
> 目前看就是未知原因导致文本转换的api请求失败了，可能要临时开debug模式看看更具体的日志给框架作者看下，我这目前是复现不出来。。

谢谢解答！请问哪个版本目前可以正常运行呢？我想切过去试试看

**Reply by wjf0214** [reply]
这个版本就可以，不过目前只有你个人出现了这个情况，不太好排查。如果方便的话，可以加 [反馈群](https://jq.qq.com/?_wv=1027&k=PXZcLlO1),进行更详细的沟通。

**Reply by ErDongChen1973** [reply]
> 这个版本就可以，不过目前只有你个人出现了这个情况，不太好排查。如果方便的话，可以加 [反馈群](https://jq.qq.com/?_wv=1027&k=PXZcLlO1),进行更详细的沟通。

好的👌🏻  谢谢！

**Comment by wokanxing** [comment]
一直是这个错误
Failed at 12/20 request,
Fail assert: {"re": "\"result\":[^0]", "from": "content"} from failed_asserts,
Request URL: https://open.e.189.cn/api/logbox/oauth2/loginSubmit.do

**Comment by weirdolx** [comment]
大佬好，我想请问一下，天翼云抽奖只抽第一次，第二次和第三次不抽了，可以更新一下嘛
登录信息：登录成功
签到状态：false，签到获得100M空间
第一次抽奖：天翼云盘50M空间
第二次抽奖： RequestFrequent
第三次抽奖： RequestFrequent

**Comment by Misuzu7** [comment]
![QQ截图20250612091709](https://github.com/user-attachments/assets/671c4a31-88c1-483d-a4a6-6daff411f011)
2025年6月12日 报错

**Reply by OldCoding** [reply]
> ![QQ截图20250612091709](https://private-user-images.githubusercontent.com/62297561/454173964-671c4a31-88c1-483d-a4a6-6daff411f011.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDk4ODYwMjgsIm5iZiI6MTc0OTg4NTcyOCwicGF0aCI6Ii82MjI5NzU2MS80NTQxNzM5NjQtNjcxYzRhMzEtODhjMS00ODNkLWE0YTYtNmRhZmY0MTFmMDExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjE0VDA3MjIwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwZGNkZmMyZDBkNTRhOWJmMWNkZmQ4NjFjMWIyYjJjODI0NzJmMzc5NjBiYzUzNzM2NDBjMDI5OGFlODVmMmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.LOkNFg0PM5j4GRLYSLaSwGhBpI7nq8-HmmzNSyzVOxY) 2025年6月12日 报错

同样的问题，6月13日开始的报错
![image](https://github.com/user-attachments/assets/0277bbb3-f372-40e2-a481-c52bc5c14532)

**Comment by OldCoding** [comment]
https://open.e.189.cn/api/logbox/oauth2/loginSubmit.do ，这个问题找到办法了，在手机上登录天翼网盘客户端，然后到设置-账号安全，关闭关闭设备锁

**Reply by crux92** [reply]
谢 兄弟 谢

**Reply by cy28438981** [reply]
我关闭了，测试还是不行，提示如图

**Reply by OldCoding** [reply]
> 我关闭了，测试还是不行，提示如图

要用安卓设备关

**Reply by Misuzu7** [reply]
可以的，设备锁默认开启，需要手动关闭

**Reply by Melody-Test** [reply]
联通关了也登陆不上..

**Reply by Ansen** [reply]
感谢, 关了可以了

**Reply by jiusun** [reply]
感谢，关闭之后可以了。并且关闭设备锁我用手机APP操作不了，可以直接通过网页进行操作。
https://e.dlife.cn/index.do

**Reply by element-chain** [reply]
感谢，关了就好了。。。但是软件设置里点关闭，关不掉，提示状态获取错误。。。结果发现得----设置-账号安全-安全中心-账号安全管理-登录-设备锁-关闭-获取验证码-关闭成功。。。

**Comment by happyzzg** [comment]
奇怪，好像又出问题了。设备锁是关闭的：

<img width="1420" height="632" alt="image" src="https://github.com/user-attachments/assets/86597287-0ffd-4ec7-849f-7dd67abaaad7" />

**Reply by foxdodo** [reply]
> 奇怪，好像又出问题了。设备锁是关闭的：
> <img alt="image" width="1420" height="632" src="https://private-user-images.githubusercontent.com/221306613/520462378-86597287-0ffd-4ec7-849f-7dd67abaaad7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjQ0ODk4ODcsIm5iZiI6MTc2NDQ4OTU4NywicGF0aCI6Ii8yMjEzMDY2MTMvNTIwNDYyMzc4LTg2NTk3Mjg3LTBmZmQtNGVjNy04NDlmLTdkZDY3YWJhYWFkNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTEzMFQwNzU5NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hOGIxZjI0ODg5NDZiMjc3YmNlNjViZDU2NDU3YjRiYjhhZjlkNTVkYzJjMTI4MzJmMmMyZDllYmUxODlkYTdjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.h9K1_TO3J4QUrLGeYz2lpUPL70pQ9ACZ9t1nyggeDA0">

https://github.com/qd-today/templates/issues/25#issuecomment-3592095876 一样，应该也是弹验证码

**Comment by happyzzg** [comment]
修改密码后，其他涉及的登录都要修改，如openlist等

**Comment by yydssdp** [comment]
修改了密码以后，又成功的运行了一段时间。最近又不行了。我刚才运行了一下，又成功了。貌似是否成功是玄学
<img width="854" height="542" alt="image" src="https://github.com/user-attachments/assets/50459e20-ade8-4bde-b4af-6ac57a7785f9" />

**Comment by siyiye** [comment]
签到挂掉了？
2026.5.4开始无法签到了
